client-cli/src/command/wallet_command.rs

```diff
@@ -25,6 +25,58 @@ impl WalletCommand {
         }
     }
 
+    fn get_mnemonics<T: WalletClient>(wallet_client: &T) -> Result<SecUtf8> {
+        let mut mnemonics: String;
+        loop {
```

Comment:
for the interactive part, I only meant `restore` mnemonics entering (instead of passing mnemonics as an argument) rather than the whole interactive part

Reply:
OK, i will change back